### ISSUE TITLE
Data quality issue with CVE-2022-0714

### ISSUE THREAD

**Issue by pxp928** [google/osv.dev#2883]
**The CVE ID**
CVE-2022-0714

**Describe the data quality issue observed**

CVE-2022-0714 is not found in OSV.dev 

This is related to the issue I opened previously: https://github.com/google/osv.dev/issues/2838/

Similar to last time, https://cve-osv-conversion.storage.googleapis.com/osv-output/CVE-2022-0714.json does exist and it is a vulnerability related to `vim/vim`.

It seems like historical CVEs are being lost periodically when the DB refreshes? 

**Suggested changes to record**
CVE-2022-0714 found in the OSV DB

**Comment by andrewpollock** [comment]
Thanks for flagging this. I didn't get a chance to review missing records, and there's ~387 records that are available for import that are missing in the export, so I'll trigger a reimport of all of these.

I've triggered a report of CVE-2022-0714 and it is now present: https://osv.dev/CVE-2022-0714

**Comment by oliverchang** [comment]
Is there a systematic issue here that we need to investigate here @andrewpollock ? Or was this from some one-off ?

**Comment by pxp928** [comment]
> Is there a systematic issue here that we need to investigate here @andrewpollock ? Or was this from some one-off ? 

Yes, I was wondering if this is going to be a recurring issue?

**Comment by andrewpollock** [comment]
It's difficult to say what happened here with certainty. 

There's a small (in the 300s) delta between CVEs available for import and CVEs **successfully** importing.

I have triggered a reimport of all of these, today, which was something I meant to do as a follow-up to https://github.com/google/osv.dev/issues/2838

Of these ~300 records, there is a further subset that fail Git range analysis due to known edge cases. When a record fails Git analysis, it fails to be imported altogether, regardless of the contents of `affected[]`. I'm working on a PR to address this failure mode at the moment.

Depending on the length of time involved between when the records were available and when they disappeared, my theory for what happened is:

- some of these CVEs originally imported successfully, because they had no GIT ranges
- when we did the clean up of what was overwritten by the Ubuntu CVE overwriting incident, they were deleted
- they then failed to reimport because a GIT range was now present

OR

- they were only present in the first place during the window of time that we erroneously imported a CVE-prefixed record from Ubuntu, and then "disappeared" again as a result of the cleanup.
